bin/tictactoe

```diff
@@ -1,3 +1,48 @@
 #!/usr/bin/env ruby
-
 require_relative '../config/environment'
+require 'pry'
```

Comment:
This is already required by the Gemfile and not needed here. If anything it also would go in `config/environment.rb` which is responsible for loading libraries and dependencies.